src/rules/export.js

```diff
@@ -87,7 +87,11 @@ module.exports = {
     return {
       'ExportDefaultDeclaration': (node) => addNamed('default', node, getParent(node)),
 
-      'ExportSpecifier': (node) => addNamed(node.exported.name, node.exported, getParent(node)),
+      'ExportSpecifier': (node) => addNamed(
+        node.exported.name,
+        node.exported,
+        getParent(node.parent)
+      ),
 
```

Comment:
i'm a bit confused; all this is doing is reformatting. do these tests pass on master?

Reply:
It changes `getParent(node)` to `getParent(node.parent)`. That would've been more obvious if I hadn't reformatted.

Reply:
Bump